frontend/public/components/utils/link.jsx

```diff
@@ -17,7 +17,7 @@ export const legalNamePattern = /[a-z0-9](?:[-a-z0-9]*[a-z0-9])?/;
 
 const basePathPattern = new RegExp(`^/?${window.SERVER_FLAGS.basePath}`);
 
-export const namespacedPrefixes = ['/search', '/status', '/k8s', '/overview', '/catalog'];
+export const namespacedPrefixes = ['/search', '/status', '/k8s', '/overview', '/catalog', '/marketplace'];
```

Comment:
We should probably not do this and keep marketplace non-namespaced.

Reply:
Why is that?

Reply:
Why does it need the namespace?

Reply:
Nvm, it's because the view of operators should not change as a function of the namespace so it does not make sense to make marketplace namespaced